constructorio-client/src/main/java/io/constructor/client/models/Task.java

```diff
@@ -32,6 +32,9 @@ public class Task {
     @SerializedName("result")
     private Object result;
 
+    @SerializedName("error")
+    private Object error;
```

Comment:
We discussed this briefly in State of Integrations. Shall we create an actual class for this?

Reply:
👍 Will do.